azure-webapp-maven-plugin/README.md

```diff
@@ -1,182 +1,175 @@
-`<stopAppDuringDeployment>` | false | To stop the target Web App or not during deployment. This will prevent deployment failure caused by IIS locking files. | 0.1.4+
-
-### V2 Configuration
-
-1. Region
-
-    The configuration `<region>` is required in V2 configuration, supported values could be found in the following.
-
-2. Runtime settings
+#### `azure-webapp:config`
+-  A command line tool providing a great experience for managing webapp deploy configuration, it will rewrite your pom config based on your input. Detail about the configuation could be found in the following.
+-  The command line will divide the config into 3 parts. 
+   1. Application
+        
+        You can config appName, resource group, region, pricing tier at this part.
+   2. Runtime
+        
+        You can config operating system and runtime stack at this part. It should be noted that you can not change the operating system after you deploy it.
+   3. DeploymentSlot
+        
+        You can config deployment slot at this part.
+
+#### `azure-webapp:deploy`
+- Deploy artifacts or docker container image to an Azure Web App based on your configuration.<br>If the specified Web App does not exist, it will be created.
+- Configuration:
+    Common configurations of all Maven Plugins for Azure can be found [here](../docs/common-configuration.md).
+
+    This maven plugin supports two kinds of configurations V2 and V1 (deprecated). Specify the configuration
+    `<schemaVersion>V2</schemaVersion>` to use the V2 configuration. Strongly suggest use v2Schema.
+    You can find v1 schema [here](v1-schema.md)
+
+    Property | Required | Description | Version
+    ---|---|---|---
+    [region](#region) | true | Specifies the region where your Web App will be hosted; the default value is **westus**. All valid regions at [Supported Regions](#region) section. | 0.1.0+
+    `<resourceGroup>` | true | Azure Resource Group for your Web App. | 0.1.0+
+    `<appName>` | true | The name of your Web App. | 0.1.0+
+    [pricingTier](#pricingtier) | false | The pricing tier for your Web App. The default value is **P1V2**.| 0.1.0+
+    `<deploymentSlot>` | false | The deployment slot to deploy your application. | 1.3.0+
+    `<appServicePlanResourceGroup>` | false | The resource group of the existing App Service Plan. If not specified, the value defined in `<resourceGroup>` will be used by default. | 1.0.0+
+    `<appServicePlanName>` | false | The name of the existing App Service Plan. | 1.0.0+
+    `<appSettings>` | false | Specifies the application settings for your Web App. You could find detail samples for appsettings at [here](../docs/web-app-samples.md). | 0.1.0+
+    `<stopAppDuringDeployment>` | false | To stop the target Web App or not during deployment. This will prevent deployment failure caused by IIS locking files. | 0.1.4+
+
+- Runtime settings
 
     Supported `<os>` values are *Linux*, *Windows* and *Docker*.
```

Comment:
The sentences here are not easy to understand, readers cannot get such plenty of information in a quick glance. Consider changing the way of expression here?